### DISCUSSION TITLE
Info

### DISCUSSION THREAD

**Discussion by sndrs** [guardian/csnx#425] [Team Posts]
If you want to get involved, come and say hi! 

[![Generic badge](https://img.shields.io/badge/google-chat-259082.svg)](https://mail.google.com/chat/u/0/#chat/space/AAAAWwBdSMs)

**Comment by jamie-lynch** [comment]
Hi! 👋

**Comment by joecowton1** [comment]
Hello 👋

**Comment by mxdvl** [comment]
👋 🧇 

Isn’t the chat room link for the old channel?
